docs/onboarding/02-digital-twin-definition-language.md

```diff
@@ -16,6 +17,7 @@ DTDL is the schema by which models are described. Its important to understand ho
 - Models are immutable. Once uploaded, they cannot be modified. Changes are made by creating a **new version**.
 - Models are versioned. The id contains the version number after the semi-colon
 - Models can be grouped/namespaced. The id can contain path segments (delimited by colon) to group models.
+- Marker tags are simply properties using a key/value Map as a schema, but they provide a more flexible way to add information to the twins at creation time (you have to define a tag in the model, but the actual values are provided when creating each twin). And you can perform queries filtering by these tags. 
 - Ontologies are peer-reviewed standardized models for common entities such as buildings, factories, and other physical spaces.
 
 ## Learning Exercises

```

Comment:
What if we also include another exercise:

### Add a marker tag
1. Modify your first model to include a marker tag.
2. Upload the updated model as a new version. 
3.  Create new twin using the updated model.
4. Update the tag value of the twin.

Reply:
Cool. I like it!

Reply:
Also added this:
5. Perform a query to get results filtering by your tag.